### PR TITLE
Bugfix/#1897 fix incorrect size of the news item's title

### PR DESCRIPTION
**Pull request by AlekOK**
(https://user-images.githubusercontent.com/51174530/111315131-c08c2f00-866a-11eb-990c-d73aa73e32a3.png)
Title is displayed and consist of 1 row and is 26px in heights.
Description is displayed and consist of 3 rows and is 69px in heights.

Expected result
Title is displayed and consist of 1 row and is 32px in heights.
Description is displayed and consist of 3 rows and is 96px in heights.

The changes correct 3 questions at once (#1889, #1897, #1891).
I corrected without mockup.

Made 4 rows in which I insert the title. Remaining lines if any remain for description.

### CODE REVIEW

**Comment by JimHolman1199** [comment]
please, change base branch do dev

**Review by JimHolman1199 — APPROVED** [review]

**Comment by JimHolman1199** [comment]
Conflicting files
src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss

**Review by KirillAbramo — APPROVED** [review]

**Review by yuriiLisunov — APPROVED** [review]

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss [hunk 1]
```diff
@@ -113,13 +113,9 @@ div {
...
   .eco-news_list-view-wrp {
     height: 204px;
-    -ms-grid-columns: 359px auto;
-    grid-template-columns: 359px auto;
-  }
-
-  .eco-news_list-content-title {
-    font-size: 21px;
-    line-height: 26px;
+ bugfix/#1897-fix-incorrect-size-of-the-news-item's-title
```
Comment (JimHolman1199) [inline src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss:116]: remove 116 line

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss [hunk 2]
```diff
@@ -113,13 +113,9 @@ div {
...
+    -ms-grid-columns: 299px auto;
+    grid-template-columns: 299px auto;
```
Comment (JimHolman1199) [inline src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss:118]: 299px, please double-check new design, so we will be sure about this values

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss [hunk 3]
```diff
@@ -128,6 +124,7 @@ div {
     height: 204px;
     grid-template-rows: 20px 96px 20px;
     word-break: break-all;
+
```
Comment (JimHolman1199) [inline src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss:127]: remove empty line

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss [hunk 4]
```diff
@@ -137,6 +134,13 @@ div {
   }
 }
 
+
```
Comment (JimHolman1199) [inline src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss:137]: extra space

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts [hunk 5]
```diff
@@ -45,17 +45,18 @@ export class NewsListListViewComponent implements AfterViewChecked {
...
 
   private getDomWidth(): string {
-    return window.innerWidth >= 1024 && window.innerWidth < 1440 ? this.smallHeight : this.bigHeight;
+    return window.innerWidth <= 768  ? this.smallHeight : this.bigHeight;
   }
 
   private getHeightOfDesc(titleHeight: number): string {
     const result = possibleDescHeight[this.getDomWidth()][titleHeight];
-    return result ? result : 'd-none';
+
```
Comment (JimHolman1199) [inline src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts:53]: extra space

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss (hunk 1 above)
Comment (AlekOK) [reply src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss:116]: done

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss (hunk 3 above)
Comment (AlekOK) [reply src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss:127]: done

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts (hunk 5 above)
Comment (AlekOK) [reply src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.ts:53]: done

**Review by idzhersh — APPROVED** [review]

File: src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss (hunk 2 above)
Comment (VitalyBorysenko) [reply src/app/component/eco-news/components/news-list/news-list-list-view/news-list-list-view.component.scss:117]: image

link to new mockup
https://www.figma.com/file/092AhCGadc1Hq8nAysxmYB/ITA-Greencity-%D0%A3%D0%91%D0%A1?node-id=12671%3A6400

**Review by JimHolman1199 — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**